bin/cocos.py

```diff
@@ -393,6 +393,10 @@ def parse_args(self, argv):
         parser.add_argument("-p", "--platform",
                             dest="platform",
                             help="select a platform (%s)" % ', '.join(platform_list))
+        parser.add_argument("--proj-dir",
```

Comment:
if check the path. this param will add to cocos.py. Because check run earlier than compile